__mocks__/api/login.ts

```diff
@@ -3,6 +3,8 @@ import { rest } from "../rest";
 export const login = [
   rest.post("/login", (_, res, ctx) =>
     res(
+      // Delays response for 2000ms.
```

Comment:
comment are code didn't match
```suggestion
      // Delays response for 1000ms.
```